composer.json

```diff
@@ -7,7 +7,7 @@
         "php": ">=8.0,<8.5"
     },
     "php-ext": {
-        "extension-name": "ext-xdebug",
+        "extension-name": "xdebug",
```

Comment:
The docs also say:

"If the extension-name is not specified, the extension-name will be derived from the Composer package name, with the vendor prefix removed. "

So maybe I should just remove that line?

Reply:
Changed